pyproject.toml

```diff
@@ -65,6 +67,8 @@ release = "python tasks/release.py"
 test = "pytest tests/"
 doc = {shell = "cd docs && mkdocs serve", help = "Start the dev server for doc preview"}
 lint = "pre-commit run --all-files"
+format = {shell = "black ./pdm ./tests && isort ./pdm ./tests && flake8 ./pdm ./tests"}
```

Comment:
It should be covered by the lint script, I integrated all linting tools into pre-commit, so this shortcut is not needed

Reply:
sorry, fixed here

da286b7